pkg/operator/csidriveroperator/driverstarter.go

```diff
@@ -152,7 +160,7 @@ func (c *CSIDriverStarterController) sync(ctx context.Context, syncCtx factory.S
 				Name:     ctrl.operatorConfig.CSIDriverName,
 			})
 			// add static assets
-			objs, err := ctrl.ctrlRelatedObjects()
+			objs, err := ctrl.getRelatedObjects()
```

Comment:
Not sure if it's worth having separate `getRelatedObjects` function, you can call `ctrl.ctrlRelatedObjects,RelatedObjects()` directly here.

Reply:
👍🏽